src/tasks/FishingTask.py

```
@@ -0,0 +1,531 @@
+        start = time.time()
+        timeout = float(self.CONTROL_TIMEOUT)
+        while time.time() - start < timeout:
+            if self.is_success_overlay():
+                self.close_success_overlay()
+                return True
+
+            state = self.get_bar_state()
+            if state is not None:
+                self.apply_bar_control(state)
+            elif time.time() - self._last_bar_log_time > 0.5:
+                self.log_info("控条阶段: 未识别到有效指针/绿区，等待下一帧")
```

Ruff (0.15.11)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)